device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/mqtt/Mqtt.java

```
@@ -127,7 +127,7 @@ void connect() throws TransportException
             }
             catch (MqttException e)
             {
-                log.warn("Exception encountered while sending MQTT CONNECT packet", e);
+                log.debug("Exception encountered while sending MQTT CONNECT packet", e);
```

Hide this exception from a too deep logging